dotCMS/src/integration-test/java/com/dotcms/datagen/VanityUrlDataGen.java

```diff
@@ -0,0 +1,108 @@
+package com.dotcms.datagen;
+
+import com.dotcms.vanityurl.business.VanityUrlAPI;
+import com.dotcms.vanityurl.model.DefaultVanityUrl;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.contentlet.model.IndexPolicy;
+import com.dotmarketing.portlets.folders.model.Folder;
+
+import io.vavr.control.Try;
+
+/**
+ * Class used to create {@link Contentlet} objects of type Persona for test purposes
+ *
+ * @author Nollymar Longa
+ */
+public class VanityUrlDataGen extends ContentletDataGen {
+
+
+
+  private String uri;
+  private String forwardTo;
+  private int action;
+  private int order;
+  private long language;
+  private String title;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field title has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)